sdcm/sct_runner.py

```diff
@@ -447,7 +447,7 @@ class AwsSctRunner(SctRunner):
     SOURCE_IMAGE_REGION = "eu-west-2"  # where the source Runner image will be created and copied to other regions
     IMAGE_BUILDER_INSTANCE_TYPE = "t3.small"
     REGULAR_TEST_INSTANCE_TYPE = "t3.large"  # 2 vcpus, 8G, 36 CPU credits/hour
-    LONGTERM_TEST_INSTANCE_TYPE = "r5.large"  # 2 vcpus, 16G
+    LONGTERM_TEST_INSTANCE_TYPE = "m7i-flex.xlarge"  # 4 vcpus, 16G
```

Comment:
Is it present on all the supported, by us, regions?

Reply:
yes, I've checked it, it's available on all region we are using